en/docs/develop-components/develop-a-web-application.md

```diff
@@ -191,6 +191,6 @@ The following limitations are specific to the Choreo Cloud Data Plane:
 
     If you encounter a blank page or a 502 error after deploying your web application, it typically indicates that the wrong directory is being served. To resolve this issue, please follow these steps:
 
-    - Double-check the build output directory, especially if you are using a Dockerfile-less preset. Ensure that the specified output directory matches the actual output directory generated during the build process. For example, make sure you enter public/ as the output directory if it is indeed public/ and not build/.
+    - Double-check the build output directory, especially if you are using a Dockerfile-less preset. Ensure that the specified output directory matches the actual output directory generated during the build process. For example, if by mistake you have entered `public/` as the output directory when it should have been `build/`.
```

Comment:
Shall we avoid the phrase `by mistake` and perhaps use a term like `erroneously` or so?